contracts/RootChain/RootChain.sol

```diff
@@ -213,7 +244,10 @@ contract RootChain {
         require(txList.length == 15);
 
         // start-exit verification
-        uint256 priority = 1000000000*txPos[0] + 10000*txPos[1] + txPos[2];
+        uint256 priority = 1000000000 * txPos[0] + 10000 * txPos[1] + txPos[2];
```

Comment:
We need to change ChallengeExit to check if input hasn't already exited, because both parent and child can be in pending state and child may maliciously choose not to challenge parent. If this happens, parent should be allowed to exit. Then people can challenge child.

Reply:
Or do not allow child to start exit until parent is successfully challenged